data/changelog/ocaml/2024-05-13-ocaml-5.2.0.md

```diff
@@ -2,6 +2,7 @@
 title: Release of OCaml 5.2.0
 description: Release of OCaml 5.2.0
 tags: [ocaml]
+versions: ["OCaml 5.2.0"]
```

Comment:
The `versions` field is used when the release name doesn't match the changelog file name or to group several releases with one changelog (eg. beta releases)

Reply:
What's the policy on prereleases? Do we want to advertise them or hide them? Aren't prerelease for the interested-in only?

Reply:
I think those should be hidden, as few people care about them and they are usually "in the know" anyways. 

But perhaps some pre-releases deserve special treatment (e.g. OCaml, dune, opam, etc).

Reply:
@Julow any views on this?

Reply:
Pre-releases may be relevant for a new "Backstage OCaml" section, I agree that on the main OCaml Changelog they aren't necessary to have